features/step_definitions/scheduler.rb

```diff
@@ -4,3 +4,23 @@
   @result = admin.cli_exec(:delete, object_type: 'priorityclasses', object_name_or_id: name)
   raise "priority classes were not deleted successfully!" unless @result[:success]
 end
+
+Given /^the #{QUOTED} scheduler CRD is restored after scenario$/ do |name|
+  ensure_admin_tagged
```

Comment:
Also need ensure_destructive_tagged as scheduler change can cause failures for other scenarios when running in parallel